script/format_sql

```diff
@@ -67,7 +67,6 @@ SKIP = {
     "sql/telemetry/fxa_users_services_first_seen/view.sql",
     "sql/telemetry/fxa_users_services_last_seen/view.sql",
     "sql/telemetry/lockwise_mobile_events_v1/view.sql",
-    "sql/telemetry/nondesktop_clients_last_seen_v1/view.sql",
```

Comment:
We allow the auto-formatter to run on this file now, so there are some whitespace changes below.